tests/test_rand.py

```diff
@@ -96,63 +112,77 @@ def test_status(self):
         # It's hard to know what it is actually going to return.  Different
         # OpenSSL random engines decide differently whether they have enough
         # entropy or not.
-        self.assertTrue(rand.status() in (1, 2))
+        assert rand.status() in (True, False)
```

Comment:
I assume it was passing before, seems to be passing now.

The [pyOpenSSL docs say](http://www.pyopenssl.org/en/stable/api/rand.html#OpenSSL.rand.status) that it returns `True`/`False`:

> Check whether the PRNG has been seeded with enough data.
> 
> Returns: `True` if the PRNG is seeded enough, `False` otherwise.

although looking [at the pyOpenSSL code](https://github.com/alexwlchan/pyopenssl/blob/master/src/OpenSSL/rand.py#L106), it seems like that error is being emitted somewhere from OpenSSL itself.

The [OpenSSL manpage](https://www.openssl.org/docs/manmaster/crypto/RAND_add.html) says:

> RAND_status() returns 1 if the PRNG has been seeded with enough data, 0 otherwise.

So I think this is several bugs in one:

* The old test was checking the wrong values
* The documentation is wrong -- `rand.status()` actually returns integers, not booleans:

    ```pycon
    >>> from OpenSSL import rand
    >>> rand.status()
    1
    ```

    It should be cast to a boolean before returning.

* We don't have a test when it doesn't have enough entropy -- all the systems have plenty of entropy, so this never returns `0`. Is there a way to drain its entropy pool for testing purposes?